tests/score_variants/test_category_score.py

```diff
@@ -0,0 +1,297 @@
+import pytest
+from typing import Union, Dict, Any
+from tempfile import NamedTemporaryFile
+from configobj import ConfigObj
+
+from genmod.score_variants import ConfigParser
+from genmod.vcf_tools import get_info_dict
+from genmod.score_variants.score_variant import get_category_score
+
+
+class ConfigObjWithNamedTemporaryFile(ConfigObj):
```

Comment:
@henrikstranneheim No there are no actual files involved. 
This is just a mocked ranking config `.ini` file residing in RAM that's only available during test execution.
The reason for adding this is because the  `ConfigParser` class only supports `.ini` files and not runtime objects.

See usage:
https://github.com/Clinical-Genomics/genmod/pull/108/files/0e977d2cac0dbb724570dcae67079a1f8fae57ca#diff-c13acf9ee59fa2c18e6d509502386156e86ad84821730c80b714de3abe09bcfdR116-R117
and 
https://github.com/Clinical-Genomics/genmod/pull/108/files/0e977d2cac0dbb724570dcae67079a1f8fae57ca#diff-c13acf9ee59fa2c18e6d509502386156e86ad84821730c80b714de3abe09bcfdR150